app/tests/spec/models/user.js

```diff
@@ -674,62 +674,80 @@ define(function (require, exports, module) {
         });
       });
 
-      describe('with an unverified account', function () {
-        beforeEach(function () {
-          account = user.initAccount({ email: 'email', uid: 'uid' });
+      // Ensure that content-server can correctly handle any emailSent response
+      var emailSentTestCases = [{
+        emailSent: undefined,
+        expectedRetryCallCount: 1
+      }, {
+        emailSent: false,
+        expectedRetryCallCount: 1
+      }, {
+        emailSent: true,
+        expectedRetryCallCount: 0
+      }];
+      emailSentTestCases.forEach(function (testCase) {
+        describe('with an unverified account, emailSent = ' + testCase.emailSent, function () {
+          beforeEach(function () {
+            account = user.initAccount({email: 'email', uid: 'uid'});
 
-          sinon.stub(account, 'signIn', function () {
-            return p();
-          });
+            sinon.stub(account, 'signIn', function () {
+              return p();
+            });
 
-          sinon.stub(account, 'get', function (property) {
-            if (property === 'verified') {
-              return false;
-            } else if (property === 'verificationReason') {
-              return VerificationReasons.SIGN_UP;
-            }
+            sinon.stub(account, 'get', function (property) {
```

Comment:
Is there a reason to use a stub and not a simple `set` on the account?

```js
account.set({
  emailSent: testCase.emailSent,
  verificationReason: VerificationReasons.SIGN_UP,
  verified: false
});
```
I don't see the stub checked for whether it's called, so I'm not sure overriding it provides any value.